# Choose the target object to grasp
target_object: "stapler"
robot_config: "gripper" #gripper // qb_hand 